core/includes/install.core.inc

```diff
@@ -1712,7 +1712,12 @@ function install_finished(&$install_state) {
   // Will also trigger indexing of profile-supplied content or feeds.
   backdrop_cron_run();
 
-  $message = st('Thank you for installing @profile!', array('@profile' => backdrop_install_profile_distribution_name()));
+  $message = st('<p>Thank you for installing @profile!</p>', array('@profile' => backdrop_install_profile_distribution_name()));
+
+  // Confirming that we are using standard profile with Dashboard. 
+  if (module_exists('dashboard')) {
+    $message .= st('<p>If you are new to Backdrop, then <a href="@dashboard">the Dashboard</a> may be a good place to start.</p>', array('@dashboard' => url('admin/dashboard')));
```

Comment:
Not sure I like the `<p>` tags, as they add a margin at the bottom whether it's needed or not... Can we do something else here instead? Maybe two `<br>`...?